src/View/Admin/WooTab.php

```diff
@@ -189,13 +204,33 @@ public function register_hooks() {
 	 */
 	public function get_sections() {
 		$sections = [
-			''                                      => esc_html__( 'Store Information', 'cc-woo' ),
-			$this->import_existing_customer_section => esc_html__( 'Import your contacts', 'cc-woo' ),
+			''  => esc_html__( 'Store Information', 'cc-woo' ),
 		];
 
 		/* This filter is documented in WooCommerce */
 		return apply_filters( 'woocommerce_get_sections_' . $this->id, $sections ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound -- Third-party hook usage.
 	}
+	/**
+	 * Add a custom wrapper for fields.
+	 *
+	 * @since  2022-06-23
+	 * @author Biplav Subedi <biplav.subedi@webdevstudios>
+	 * @return string
+	 */
+	public function add_optional_fields_wrapper() {
+		echo "<tbody id='cc-optional-fields'>";
+	}
+
+	/**
+	 * Add a custom wrapper for fields end.
+	 *
+	 * @since  2022-06-23
+	 * @author Biplav Subedi <biplav.subedi@webdevstudios>
+	 * @return string
+	 */
+	public function add_optional_fields_wrapper_end() {
+		echo "</tbody>";
+	}
```

Comment:
Let's look into changing the since from a date to the intended version for this PR. If we're not sure exactly which version that'd be, then either the `??` or `TBD`